taxi/tests/test_models.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car
+
+
+class ModelsTests(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(name="test", country="USA")
+        self.assertEqual(str(manufacturer),
+                         f"{manufacturer.name}, {manufacturer.country}")
+
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(
+            username="test",
+            password="test123",
+            first_name="test_first",
+            last_name="test_last",
+        )
+        self.assertEqual(
+            str(driver),
+            f"{driver.username} ({driver.first_name} {driver.last_name})")
```

Comment:
In case there are no arguments on the same line with an opening parenthesis, your closing parenthesis should not have any arguments on the same line as well. Please, resolve this issue